docker/appserver/Tomcat/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG TOMCAT_VERSION=8.5.60-jdk8
```

Comment:
moet dit niet jre zijn?

Reply:
Daar zou je dan het Tomcat Docker team op aan moeten spreken, ze doen het nu met jdk.

Reply:
tomcat:8.5.41-jre8 bestaat

Reply:
op https://hub.docker.com/_/tomcat is er geen enkele versie met jre, ze zijn allemaal met jdk

Reply:
En bovendien: je hebt een JDK nodig om .jsp te kunnen compileren, toch?

Reply:
See: https://github.com/docker-library/openjdk/issues/272 after Java 8, Oracle stopped publishing the JRE artifacts.
As for the JSP's, Tomcat uses Jasper to compile them instead of the JRE, see: http://tomcat.apache.org/tomcat-5.5-doc/jasper-howto.html